api/printful/printful.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+require(BASE . '/api/api-cache.php');

```

Comment:
Is it a guarantee that `BASE` is defined here? I think you might have just been lucky that `BASE` is also provided upstream, and you should actually be importing the template/top first here.
FYI, php will only re-import the template/top here if it wasn't already imported upstream.